modules/rte_mis_core/src/Plugin/Block/SiteMenuTextSectionBlock.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Drupal\rte_mis_core\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Provides a 'Site Menu Text Section' block with image and text fields.
+ *
+ * @Block(
+ *   id = "site__menu_text_section_block",
+ *   admin_label = @Translation("Site Menu Text Header & Footer Block"),
+ * )
+ */
+class SiteMenuTextSectionBlock extends BlockBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    $config = $this->getConfiguration();
+    $build = [];
+
+    $currentYear = date('Y');
+    $text = isset($config['block_text']) && $config['block_text'] !== '' ? $config['block_text'] : "Copyright © $currentYear. Department of School and Mass Education, Government of India. All Rights Reserved.";
+
+    $build['sub_heading']['#markup'] = '<p>' . $text . '</p>';
+
+    return $build;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function blockForm($form, FormStateInterface $form_state) {
+    $form = parent::blockForm($form, $form_state);
+    $currentYear = date('Y');
+    $config = $this->getConfiguration();
+
+    $form['block_text'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Block Text'),
+      '#default_value' => isset($config['block_text']) && $config['block_text'] !== '' ? $config['block_text'] : "Copyright © $currentYear. Department of School and Mass Education, Government of India. All Rights Reserved.",
```

Comment:
Same here - Could you keep the static string wrapped inside t()?  Also, the year used will be static once the user save the block config. See the file - `config/install/block.block.gin_sitemenutextheaderfooterblock_2.yml`. The year is kept static to 2024.

Reply:
done.